api/config/config_stable.json

```diff
@@ -111,7 +111,7 @@
   },
   "featureFlags": {
     "unsafeAllowDeleteUser": false,
-    "unsafeAllowAccessToAllTiersForRegisteredUsers": false,
+    "unsafeAllowAccessToAllTiersForRegisteredUsers": true,
```

Comment:
nit: our goal with stable is to mirror prod as closely as possible (without interacting with prod data), so I think there's a good argument to leave this one false. This flag is unfortunately a bit of an oddball as we never want it to hit prod - it's hard for me to reason about what the correct value should be in lower environments. test enablement feels correct, but I think I could be convinced either way on perf/staging (I don't object to enabling there).

Reply:
OK.  We can best match environments this way.

Test, Local, Perf, Staging: fully set up fake-Controlled Tier
Preprod: fully set up real Controlled Tier
Stable and Prod: create projects and perimeters only